core/src/main/java/org/bouncycastle/util/Properties.java

```diff
@@ -115,6 +115,31 @@ public static boolean removeThreadOverride(String propertyName)
         return false;
     }
 
+    /**
+     * Return propertyName as an integer, defaultValue used if not defined.
+     *
+     * @param propertyName name of property.
+     * @param defaultValue integer to return if property not defined.
+     * @return value of property, or default if not found, as an int.
+     */
+    public static int asInteger(String propertyName, int defaultValue)
```

Comment:
These changes were introduced in the original commit [e7e4d3e](https://github.com/lineaje-labs-gos/bc-java/commit/e7e4d3e978cafca9df665edba740954950296d62)

Reply:
Correct, but we just need the function as its part of the fix and doesn't affect anything else.